manifests/grafana-deployment.yaml

```diff
@@ -180,9 +178,6 @@ spec:
       - configMap:
           name: grafana-dashboard-pod-total
         name: grafana-dashboard-pod-total
-      - configMap:
-          name: grafana-dashboard-pods
-        name: grafana-dashboard-pods
```

Comment:
@paulfantom Can you please share what was the reason of removing 'pods' dashboard?

Reply:
That was a year ago...

Possibly [kubernetes-mixin](https://github.com/kubernetes-monitoring/kubernetes-mixin) project removed it.

Reply:
This is the PR in which the dashboard is removed in kubernetes-mixin:
https://github.com/kubernetes-monitoring/kubernetes-mixin/pull/362/files